spec/services/articles/updater_spec.rb

```diff
@@ -95,6 +95,48 @@
         expect(Mentions::CreateAll).not_to have_received(:call).with(article)
       end
     end
+
+    context "when an article is unpublished and contains comments" do
```

Comment:
These new specs test functionality that _already existed_ prior to any of my changes, but was never tested: 

https://github.com/forem/forem/blob/e53b6bff853a5f79bbebbf1ce8be98eae6a0b024/app/services/articles/updater.rb#L46-L49